src/styl/site.styl

```diff
@@ -153,6 +153,12 @@ font-sizes($font-size-basic, $font-size-extra)
 
 @media screen and (min-width: 1024px)
 
+  $gutter = 21px
+
+  grid-maker(680px, $gutter, ml)
```

Comment:
if we gonna do this I think it just has to become `m` and `l` becomes `xl` etc

Reply:
ok i guess thats fine here cuase we dont have so much markup

Reply:
idk i think its better how it is, with `ml`.  thats less confusing to me somehow